content/tutorials/quickstart/transfer-nftokens.md

```diff
@@ -175,34 +169,59 @@ Connect to the ledger and get the wallet accounts.
 
 
 ```
-      async function createSellOffer() {
-        const standby_wallet = xrpl.Wallet.fromSeed(standbySeedField.value)
-        const operational_wallet = xrpl.Wallet.fromSeed(operationalSeedField.value)
-          let net = getNet()
-          const client = new xrpl.Client(net)
-          results = 'Connecting to ' + getNet() + '...'
-          document.getElementById('standbyResultField').value = results
-        await client.connect()
-          results += '\nConnected. Creating sell offer...'
-          document.getElementById('standbyResultField').value = results
+async function createSellOffer() {
+  const standby_wallet = xrpl.Wallet.fromSeed(standbySeedField.value)
+  const operational_wallet = xrpl.Wallet.fromSeed(operationalSeedField.value)
+  let net = getNet()
+  const client = new xrpl.Client(net)
+  results = 'Connecting to ' + getNet() + '...'
+  document.getElementById('standbyResultField').value = results
+  await client.connect()
+  results += '\nConnected. Creating sell offer...'
+  document.getElementById('standbyResultField').value = results
+```
 
+Compute the Expiration Date, if present. The expiration date represents the number of seconds after the Ripple Epoch that the offer should expire. Subtract the Ripple Epoch from the current system date divided by 1,000, and add the number of days (or fractions of a day) multiplied by 86400 to get the correct expiration date.
 
+```
+  //------------------------------------- Prepare Expiration Date
+  var expirationDate = null
+  if (standbyExpirationField.value !="") {
+    var days = document.getElementById('standbyExpirationField').value
+    var secondsInDay = 86400
+    var rippleEpoch = 946684800
+    var currentTimestamp = (new Date() / 1000) - rippleEpoch
+    expirationDate = parseInt(Math.floor(currentTimestamp + (days*secondsInDay)))
```

Comment:
Rather than figuring out tricky format conversions on their own, we should encourage users to rely on the conversion functions that are built into xrpl.js. For example:

```suggestion
    let d = new Date()
    d.setDate(d.getDate() + parseInt(days))
    var expirationDate = xrpl.isoTimeToRippleTime(d)
```

(The difference between `let` and `var` is a separate thing, which is probably worth discussing at some point, but not necessary to fix at this time)